src/components/dev-dot-content/mdx-components/mdx-alert/index.tsx

```diff
@@ -0,0 +1,10 @@
+import Alert, { AlertProps } from 'components/alert'
+import s from './mdx-alert.module.css'
+
+export function MdxAlert(props: AlertProps) {
```

Comment:
I chose to compose the final 'mdx' version of the alert, in case we want to use the base component elsewhere in the future.